packages/ui/tailwind.config.js

```diff
@@ -1,6 +1,13 @@
-const sharedConfig = require('tailwind-config/tailwind.config.js');
+import sharedConfig from 'tailwind-config/tailwind.config.js';
 
 module.exports = {
   content: ['./components/**/*.tsx'],
   presets: [sharedConfig],
+  theme: {
+    extend: {
+      boxShadow: {
+        'service-card': '0px 0px 29px 0px rgba(190, 24, 93, 0.50)',
```

Comment:
Do not tie CSS styles to its usage. Instead, use a name that represents what the styles do/are:

```suggestion
        'active-shadow': '0px 0px 29px 0px rgba(190, 24, 93, 0.50)',
```